features/number.yml

```
@@ -0,0 +1,85 @@
+name: Number
+description: The number type (and `Number` object) represents floating-point numbers, such as 42 or -4.201. JavaScript numbers can also represent boundless negative and positive values as `Infinity` or not-a-number as `NaN` (as in `0 * Infinity`).
```

I admit it _does_ feel like I'm shoehorning a lot in here with this description, but we can always revise later. 😄 

```suggestion
description: The number type (and `Number` object) represents floating-point numbers, such as 42 or -4.201, while the `Math` API contains mathematical functions and constants. JavaScript can also represent boundless negative and positive values as `Infinity` or not-a-number as `NaN` (as in `0 * Infinity`).
```